RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronSequence_cff.py

```diff
@@ -0,0 +1,88 @@
+import FWCore.ParameterSet.Config as cms
+
+#==============================================================================
+# Sequence to make low pT electrons.
+# Several steps are cloned and modified to 'open up' the sequence:
+# tracker-driven electron seeds, KF track candidates, GSF tracks.
+#==============================================================================
+
+# Low pT electron seeds
+# Below relies on default configuration for generalTracks
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeeds_cfi import *
```

Comment:
@mverzett We can read in pfTracks instead of generalTracks and match to both ECAL and HCAL clusters using your simpler logic, here: https://github.com/bainbrid/LowPtElectrons/blob/LowPtElectrons_prod/plugins/TrackerElectronsFeatures.cc#L355-L382
We'd have to schedule a PFTrackProducer module between generalTracks and lowPtElectronSeeds, and reset the gsfTrack InputTag to a null value? (https://github.com/cms-sw/cmssw/blob/master/RecoParticleFlow/PFTracking/python/pfTrack_cfi.py#L6). I'll revisit later ...